orbit/pkg/augeas/augeas_unix.go

```diff
@@ -0,0 +1,55 @@
+//go:build !windows
+// +build !windows
+
+package augeas
+
+import (
+	"embed"
+	"io"
+	"os"
+	"path/filepath"
+)
+
+//go:embed lenses
+var lenses embed.FS
+
+func CopyLenses(installPath string) (string, error) {
+	outPath := filepath.Join(installPath, "lenses")
+
+	err := os.RemoveAll(outPath)
+	if err != nil {
+		return "", err
+	}
+	err = os.MkdirAll(outPath, 0o755)
+	if err != nil {
+		return "", err
+	}
+	entries, err := lenses.ReadDir("lenses")
+	if err != nil {
+		return "", err
+	}
+	for _, entry := range entries {
+		src, err := lenses.Open(filepath.Join("lenses", entry.Name()))
+		if err != nil {
+			return "", err
+		}
+		dest, err := os.OpenFile(filepath.Join(outPath, entry.Name()), os.O_CREATE|os.O_WRONLY, 0o644)
+		if err != nil {
+			return "", err
+		}
+		_, err = io.Copy(dest, src)
+		if err != nil {
+			return "", err
+		}
+		src.Close()
+		if err != nil {
+			return "", err
+		}
+		dest.Close()
+		if err != nil {
```

Comment:
Did you mean to use `err := dest.Close()` here?

Reply:
Yes, oops. I'm not sure if this error handling is necessary though, since it's common to defer the close and AIUI the error condition there isn't handled

Reply:
Totally! We just about to enable the `errcheck` linter though so these will end up throwing a lint error if we don't handle these (or explicitly exclude from the linter).

Reply:
Okay, that sounds good, fixed